examples/resources/indykite_application_space_configuration/resource.tf

```diff
@@ -0,0 +1,22 @@
+resource "indykite_application_space_configuration" "development" {
+    app_space_id = "gid:app-space-gid"
+
+    default_auth_flow_id = "gid:auth-flow-gid"
+    default_email_service_id = "gid:email-gid"
+    default_tenant_id = "gid:tenant-gid"
+
+    username_policy {
+        allowed_username_formats = ["email"] # Possible also username and mobile
+        valid_email = true
+        verify_email = true
+        verify_email_grace_period = "600s"
+        allowed_email_domains = ["gmail.com", "outlook.com"]
+        exclusive_email_domains = ["indykite.com"]
+    }
+    unique_property_constraints = {
+        # Need to specify as JSON, currently only tenantUnique and canonicalization is supported.
+        # canonicalization can contains 'unicode' or 'case-insensitive'.
+        "property1": jsonencode({"tenantUnique": false})
```

Comment:
After "property1" should be sign "=" not ":"

Reply:
Based on documentation, both is allowed https://developer.hashicorp.com/terraform/language/expressions/type-constraints

Do you think I should use `=` rather?

Reply:
Well, I stick with =, because it is common for every field in terraform and someone could think that is error. Up to you :)